File: 1134.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var alcool = 0; var gasolina = 0; var diesel = 0;
while(true){
    var n = Number(lines.shift());
    if(n === 4){
        break;}
    else if(n === 3){
        var diesel = diesel + 1}
    else if(n === 2){
        var gasolina = gasolina + 1}
    else if(n === 1){
        var alcool = alcool + 1}}
console.log(`MUITO OBRIGADO
Alcool: ${alcool}
Gasolina: ${gasolina}
Diesel: ${diesel}`);

File: 1134.cpp
#include <iostream>
#include <iomanip>
using namespace std;
int main(){
    int alcool, gasolina, diesel;
    alcool = gasolina = diesel = 0;
    while(true){
        int n;
        cin >> n;
        if(n == 4){
            break;
        }
        if(n == 3){
            diesel += 1;
        }
        if(n == 2){
            gasolina += 1;
        }
        if(n == 1){
            alcool += 1;
        }
    }
    cout << "MUITO OBRIGADO\nAlcool: " << alcool << "\nGasolina: " << gasolina << "\nDiesel: " << diesel << endl;
}
